src/views/components/display-html/DisplayHTML.native.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import Html from 'react-native-render-html';
+import { string } from 'prop-types';
+import { Box } from '../../components';
+
+class DisplayHTML extends Component {
+  static defaultProps = {
+    content: '',
+  }
+
+  static propTypes = { 
+    content: string,
+  };
+
+  render() { 
+    const { content } = this.props;
+
+    return (
+      <Box>
```

Comment:
I don't think we need to wrap this inside of a Box, as it will block us from adding styling deeper into this component where we may need in the future.